[Feat] New In-Band Peering (#2720)
liqotech/liqo: docs/features/peering.md

```

```{warning}
The user can adopt different peering approaches depending if it has contemporary access to both clusters or not, as described in the [dedicated page](/advanced/peering-strategies.md). 

> If the user can not expose the API server to the public, there is the possibility to leverage the Liqo networking to let the API server traffic flow into the cross-cluster VPN tunnel by leveraging the [IP remapping](../advanced/external-ip-remapping.md) (of the `liqo-proxy` service) and the [API Server proxy](../advanced/api-server-proxy.md) features.

Reviewer: ```suggestion
If the user cannot expose the API server to the public, you can leverage the Liqo networking to let the API server traffic flow into the cross-cluster VPN tunnel by leveraging the [IP remapping](../advanced/external-ip-remapping.md) (of the `liqo-proxy` service) and the [API Server proxy](../advanced/api-server-proxy.md) features.
```

Reviewer: I'm not sure I would understand how to configure this feature. Would it be possible to provide an example?

Author: In a default installation, the only required step is to add a `--in-band` flag to liqoctl peer or liqoctl authenticate commands, as explained in the following sentence.

> If you cannot expose the Kubernetes API Server to the public, you can leverage the Liqo networking to let the API server traffic flow into the cross-cluster VPN tunnel by leveraging the [IP remapping](../advanced/external-ip-remapping.md) (of the `liqo-proxy` service) and the [Kubernetes API Server proxy](../advanced/k8s-api-server-proxy.md) features.

Reviewer: ```suggestion
If you cannot expose the Kubernetes API Server to the public, you can leverage the Liqo networking to connect to the remote API server, and the related traffic will flow into the cross-cluster VPN tunnel by leveraging the [IP remapping](../advanced/external-ip-remapping.md) (of the `liqo-proxy` service) and the [Kubernetes API Server proxy](../advanced/k8s-api-server-proxy.md) features.
```

> You can achieve this by setting the `--in-band` flag in the `liqoctl peer` or `liqoctl authenticate` commands.

Reviewer: ```suggestion
You can achieve this by setting the `--in-band` flag in the `liqoctl peer` or `liqoctl authenticate` commands, which automatically configure all the features needed for this mechanism to work (i.e., the API server proxy and the IP remapping).
```

> Note that the networking module must be enabled.

Reviewer: ```suggestion
Note that, for this feature to work, the Liqo networking module must be enabled.
```
